packages/react-ai/package.json

```diff
@@ -52,7 +52,7 @@
     "@aws-amplify/ui-react-core": "3.0.19"
   },
   "devDependencies": {
-    "@aws-amplify/data-schema": "0.0.0-ai-20240815192737",
+    "@aws-amplify/data-schema": "^1.5.0",
```

Comment:
Does _@aws-amplify/data-schema_ need to be listed as a peer dep?

Reply:
Hm actually probably not anymore since we don't need to override it, checking

Reply:
removed